x-pack/plugins/ml/public/application/explorer/anomaly_timeline.tsx

```diff
@@ -156,6 +157,16 @@ export const AnomalyTimeline: FC<AnomalyTimelineProps> = React.memo(
                     />
                   </EuiFormRow>
                 </EuiFlexItem>
+                {selectedCells ? (
+                  <EuiFlexItem grow={false}>
+                    <EuiButtonEmpty size="xs" onClick={setSelectedCells.bind(null, undefined)}>
+                      <FormattedMessage
+                        id="xpack.ml.explorer.clearSelectionLabel"
+                        defaultMessage="Clear selection"
```

Comment:
I think we also need a way to clear the selection when the swim lane is used as an embeddable.